api/src/test/java/org/openmrs/api/VisitServiceTest.java

```diff
@@ -13,6 +13,11 @@
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertTrue;
+import static org.hamcrest.MatcherAssert.assertThat;
```

Comment:
we sort static imports alphabetically, the `org.hamcrest` comes before `org.junit`.

not sure which IDE you use, but we have a set of styling rules we try to follow, you can configure your IDE according to https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE

Reply:
@teleivo thanks. I will make tha changes. thanks